tool-openssl/x509_test.cc

```diff
@@ -325,6 +430,73 @@ TEST_F(X509ComparisonTest, X509ToolCompareModulusOpenSSL) {
   ASSERT_EQ(tool_output_str, openssl_output_str);
 }
 
+// Test against OpenSSL output "openssl x509 -in file -subject"
+TEST_F(X509ComparisonTest, X509ToolCompareSubjectOpenSSL) {
```

Comment:
In a follow up PR can you please refactor all of these tests into a struct that takes the command to run and a cleanup function and then just loops over all of them.